### DISCUSSION TITLE
More on the Project Reunion approach and roadmap!

### DISCUSSION THREAD

**Discussion by mevey** [microsoft/WindowsAppSDK#369] [Announcements]
You asked for more information on Project Reunion, so we've added more detailed [documentation](https://github.com/microsoft/ProjectReunion/blob/master/docs/README.md) explaining:

- what Project Reunion is and what it is not, 
- how we are approaching the work and 
- what is coming.

Read all about it here: https://github.com/microsoft/ProjectReunion/blob/master/docs/README.md

As always we want to hear from you. Does this makes sense? Do you have concerns and questions? You can respond to this thread or [start new issue threads](https://github.com/microsoft/ProjectReunion/issues/new/choose) as you see fit. We will keep sharing more information with you: design docs, samples and documentation as we embark on this journey towards evolving the Windows app platform.

Stay tuned.

**Comment by hansmbakker** [comment]
This is a great start already!

I can imagine the project is in "requirements gathering phase" but it would be very nice if it could be made more concrete:
- https://github.com/microsoft/ProjectReunion/blob/master/docs/roadmap.md is still very high level
- some explanation about the development / release approach: whether things will be released in bigger chunks when they are complete or whether smaller pieces of functionality will be delivered in an agile / iterative way
- maybe issues visualized on a [project board](https://github.com/microsoft/ProjectReunion/projects) or linked to milestones could give better insight in what people can expect

**Comment by pjmlp** [comment]
Yes it looks quite good.

However having seen similar material from Microsoft since Windows 8, introduction, I feel there are certain areas that need to be improved still, and that all projects that are part of Project Reunion should actually act as one, instead of making us jump from silo to silo with our feedbacks, ideas, contributions, ...

From the roadmap description I think it focus too much on the libraries, but says too little about the tooling story, which is well known how much yak shaving it has required with 8 => 8.1 UAP => UWP transitions, and now once again with UWP => Reunion.

Or replacing the nice C++/CX tooling with C++/WinRT manual editing of IDL files, and reading through all documentation and Stack Overflow answers how to port code into C++/WinRT.

Also how C#/WinRT plays with .NET Native no longer being around, eventually, I guess it depends on where the Wind blows with .NET.

So while one might be able to use a new set of common libraries, the overall picture seems to be missing what functionality is actually going to exist and what is being dropped without any upgrade path (WPF features vs WinUI 3.0 capabilities).

Specially relevant as someone that thought WinRT was what .NET 1.0 should have been, now with Project Reunion there is feeling, "_lets just use Win32, MFC and .NET as we always did_", so a message is required for those developers that never cared about UWP to take a second look at Project Reunion and actually adopt some of those technologies.

For those of us with experience in other platforms, Project Reunion message also has certain resemblances with Android's Android Support Library, nowadays known as Android Jetpack, so also a possible source of inspiration how to spread the project's message.

All in all I am positive about the project goal's, the roadmap update is also interesting to read, and look forward to have all Windows development stacks under the same roof.

Sorry if my comment isn't the kind of feedback you were looking for.

**Comment by Poopooracoocoo** [comment]
Will it also include some info on #82?

@jonwis said
>Project Reunion will not alter the core Win32 controls in comctl32.dll that are part of Windows and used by existing applications. As others have noted, such a change would be challenging to pull off across the breadth of the Windows app ecosystem.

Doesn't sound right AT ALL, considering how Microsoft has changed it in every release of Windows, including the first few builds of Windows 10. 

>If you’d like to use modern style XAML-based controls in your Win32 app please keep an eye on WinUI 3. You’ll be able to drop a WinUI 3 control into your existing applications to start applying your own theming and branding to them.

This response is hilarious if it's not sarcastic. I'm here because I want my Windows experience to improve. I want Microsoft products to be better. Microsoft isn't using WinUI, even in brand new apps like Skype, Teams and Edge (Chromium). If you really believe that, you've got some serious issues and need to take some time off work and work on them.

**Comment by Poopooracoocoo** [comment]
And is there anything on access to virtual desktops and the system media transport controls? at times I'm tempted to go back to Windows 10 v1709, where task view was a delight to use. There's tools like SylphyHornEx that have to really dig to hook into virtual desktops.
